library/block/block_content_registry/src/testmod/resources/data/quilt_block_content_registry/attachments/minecraft/block/flattenable_block.json

```diff
@@ -0,0 +1,8 @@
+{
+  "replace": false,
+  "values": {
+    "minecraft:oak_planks": {
+      "Name": "minecraft:oak_slab"
```

Comment:
eww, caps on a field
i just wanted to point this out, although yeah, I don't think there's a better solution

Reply:
Minecrafts codec is fun. Don't like it either.